### ISSUE TITLE
Latest master failed to compile in Notebook

### ISSUE THREAD

**Issue by ProfFan** [borglab/SwiftFusion#185]
```swift
%install '.package(url: "https://github.com/borglab/SwiftFusion", .branch("5eef7be3eece5a5c9b508077b74c5f0ea8ba46c5"))' BeeDataset SwiftFusion
```
```swift
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Core/TrappingDouble.swift:98:32: error: use of undeclared type 'XLAScalarWrapper'
  public var xlaScalarWrapper: XLAScalarWrapper { return value.xlaScalarWrapper }
                               ^~~~~~~~~~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Core/TrappingDouble.swift:99:79: error: type 'Double' has no member 'xlaTensorScalarTypeRawValue'
  public static var xlaTensorScalarTypeRawValue: UInt32 { return Swift.Double.xlaTensorScalarTypeRawValue }
                                                                 ~~~~~~~~~~~~ ^~~~~~~~~~~~~~~~~~~~~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:205:14: error: use of undeclared type 'AnyValue'
    _ self_: AnyValue,
             ^~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:210:20: error: use of undeclared type 'AnyValue'
    _ self_: inout AnyValue,
                   ^~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:215:20: error: use of undeclared type 'AnyValue'
    _ self_: inout AnyValue,
                   ^~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:220:20: error: use of undeclared type 'AnyValue'
    _ self_: inout AnyValue,
                   ^~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:231:14: error: use of undeclared type 'AnyValue'
    _ self_: AnyValue,
             ^~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:239:20: error: use of undeclared type 'AnyValue'
    _ self_: inout AnyValue,
                   ^~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:247:20: error: use of undeclared type 'AnyValue'
    _ self_: inout AnyValue,
                   ^~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:255:20: error: use of undeclared type 'AnyValue'
    _ self_: inout AnyValue,
                   ^~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:268:23: error: use of undeclared type 'AnyValue'
  typealias Storage = AnyValue
                      ^~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:316:12: error: type of expression is ambiguous without more context
    vtable = demandVTable(.init(Type<Self>.id, Type<C>.id)) {
    ~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:394:18: error: cannot infer contextual base in reference to member 'init'
      storage = .init(x)
                ~^~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:411:39: error: type of expression is ambiguous without more context
    try vtable[0].existentialDispatch.withContiguousStorageIfAvailable(storage) {
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:443:18: error: cannot infer contextual base in reference to member 'init'
      storage = .init(x)
                ~^~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:503:39: error: type of expression is ambiguous without more context
    try vtable[0].existentialDispatch.withUnsafeMutableBufferPointerIfSupported(&storage) {
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:515:39: error: type of expression is ambiguous without more context
    try vtable[0].existentialDispatch.withContiguousMutableStorageIfAvailable(&storage) {
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/workspaces/SwiftFusion/Notebooks/swift-install2/package/.build/checkouts/SwiftFusion/Sources/SwiftFusion/Inference/PenguinExtensions.swift:532:39: error: type of expression is ambiguous without more context
    try vtable[0].existentialDispatch.partition(&storage, &r) { p in
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
Install Error: swift-build returned nonzero exit code 1.
```

@marcrasi Appears to be a breaking change in Penguin?

**Comment by ProfFan** [comment]
Strange, can only reproduce with nightly toolchain...

**Comment by ProfFan** [comment]
Also when running the `Bee Tracking` colab:
```
[I 03:32:50.505 NotebookApp] proxying WebSocket connection to: ws://localhost:8888/api/kernels/a1c3b45f-7f72-4fa6-a68e-b8373be6e013/channels?session_id=cd4db44e643142f7c847bfb4c7cbcffd&jupyter_http_over_ws_auth_url=http%3A%2F%2Flocalhost%3A8888%2F%3Ftoken%3Dswift-token-0x34f
clang: CommandLine Error: Option 'h' registered more than once!
LLVM ERROR: inconsistency in registered CommandLine options
[I 03:35:05.237 NotebookApp] KernelRestarter: restarting kernel (1/5), keep random ports
WARNING:root:kernel a1c3b45f-7f72-4fa6-a68e-b8373be6e013 restarted
```

**Comment by marcrasi** [comment]
Looking!

**Comment by marcrasi** [comment]
The bee notebook (https://colab.sandbox.google.com/drive/18uJvXwxSrahnqv16dVEwt66_qaOZVgp1) works for me, on both the latest nightly toolchain and the stable 0.11 release. (But not the tracking section, which looks like it hasn't been updated for the new bee videos yet).

Does it still break for you? Maybe try deleting the `swift-install2` cache? Maybe some dependency had a problem that got fixed between when you posted and when I tried.
